controllers/CommentController.js

```diff
@@ -23,6 +26,17 @@ class CommentController {
         articleId,
         content
       });
+      // event emitter
```

Comment:
Hi @akpante3. Nice job, Kindly remove this comment and all other unnecessary comments. Cheers.